integrationTests/cypress/integration/redmine_administration.feature

```diff
@@ -0,0 +1,69 @@
+Feature: Administrative Procedures
+
+  @requires_testuser
+  Scenario: cas user + no redmine user => login => create normal redmine account
+    Given the user is not member of the admin user group
+    And the user has no internal redmine account
+    And the user is logged out of the CES
+    When the user logs into the CES
+    Then the user has an internal redmine account with default privileges
+
+  @requires_testuser
+  Scenario: cas user (admin) + no redmine user => login => create admin redmine account
```

Comment:
Suggestion:
"cas user (admin) + no redmine user => login => create admin redmine account" -->
"An admin CAS user logs into redmine for the first time and consequently an admin redmine account is created"